docs/interactions/structured-messages/webview-sdk.md

```diff
@@ -0,0 +1,102 @@
+# About Webview SDK
+
+This SDK is meant to be used on pages linked to by messages/items with a "webview" target.
+
+This kind of target is useful to create specific interactions that are not provided by channels.
+
+The Webview SDK provides a generic way to interact with supported channels that can open your page in a webview which allows you to use the same target on different channels without having to write multiple integrations.
+
+## Supported channels
+
+The SDK supports the following channels:
+
+- Engage Messaging (Web only for now)
+
+## Integration
+
+You can load the SDK by adding the following `script` tag to your target page.
+You'll need to replace `<YOUR_DOMAIN>` with your ED domain's name.
+
+```html
+<script>
+  (function (d, s, url, id) {
+    d.addEventListener('DOMContentLoaded', function() {
+      var js, fjs = d.getElementsByTagName(s)[0];
+
+      if (d.getElementById(id)) return;
+
+      js = d.createElement(s);
+      js.id = id;
+      js.src = url;
+      js.async = true;
+      fjs.parentNode.insertBefore(js, fjs);
+    });
+  })(document, 'scr' + 'ipt', 'https://<YOUR_DOMAIN>.engagement.dimelo.com/webview/sdk.js', 'ed_webview_sdk_loader');
```

Comment:
We should also put the hostname in a variable. Because we can also have digital.ringcentral.com

Reply:
Changed in [that commit](https://github.com/ringcentral/engage-digital-api-docs/pull/43/commits/baae12e71572aeb36b1142f8ecff70a174811d5c).